### ISSUE TITLE
SOM Training: suspiciously low validation error

### ISSUE THREAD

**Issue by PetrToman** [encog/encog-java-core#82]
1. In Workbench, create .ega file for SOM classification - using data: http://dione.zcu.cz/~toman40/encog/data7.zip - and set "Maximum Error Percent (0-100)" = 10.
2. Excecute "task-full".
3. Create `data7_eval.egb` by "Normalize to Training(EGB)" action.
4. Double click `data7_train.eg`, click "Train", set "Training Set" = data7_train.egb and "Validation Set (optional)" = data7_eva.egb and select "Basic... Training".
5. Run the training again -> "Validation Error" will be ~7.5 %, but there is only ~50 % of correctly classified rows in `data7_output.csv` (where `y`=`Output:y`).
